include/caffe/data_transformer.hpp

```diff
@@ -23,7 +23,7 @@ class DataTransformer {
    * @brief Initialize the Random number generations if needed by the
    *    transformation.
    */
-  void InitRand();
+  void InitRand(unsigned int seed);
```

Comment:
Can we avoid most of the InitRand call changes by keeping the argument-less version and invoking `InitRand(caffe_rng_rand())` in that case?